src/utils/use-twitch-api.js

```diff
@@ -1,13 +1,11 @@
 import { useState, useEffect, useCallback } from 'react';
+import fetchTwitchVideos from '../utils/fetch-twitch-videos';
 import { get } from './request';
-
-const TWITCH_API_ENDPOINT = 'https://api.twitch.tv/helix/';
-const CLIENT_ID = '041r2glmgub2pt357ss0la44j2sz95';
-const MDB_CHANNEL_ID = '467752938';
-const STREAMS_URL = `${TWITCH_API_ENDPOINT}streams?user_id=${MDB_CHANNEL_ID}`;
-const video_URL = `${TWITCH_API_ENDPOINT}videos?user_id=${MDB_CHANNEL_ID}&first=`;
-
-const headers = { 'Client-ID': CLIENT_ID };
+import {
+    TWITCH_headers,
+    TWITCH_STREAMS_URL,
+    TWITCH_VIDEO_URL,
+} from '../constants';
```

Comment:
Nice! Yeah I think having these constants more easily accessible from both use cases in a separate file makes sense.